lib/mayaUsd/utils/diagnosticDelegate.cpp

```diff
@@ -67,13 +67,17 @@ class _WarningOnlyDelegate : public UsdUtilsCoalescingDiagnosticDelegate
 
 static MString _FormatDiagnostic(const TfDiagnosticBase& d)
 {
+    // strip out all the characters before "maya-usd"
+    std::string filterFilePath { d.GetContext().GetFile() };
+    filterFilePath = filterFilePath.substr(filterFilePath.find("maya-usd"));
+
     const std::string msg = TfStringPrintf(
         "%s -- %s in %s at line %zu of %s",
         d.GetCommentary().c_str(),
         TfDiagnosticMgr::GetCodeName(d.GetDiagnosticCode()).c_str(),
         d.GetContext().GetFunction(),
         d.GetContext().GetLine(),
-        d.GetContext().GetFile());
+        filterFilePath.c_str());
```

Comment:
filter the file path so it discards everything before the project name (`maya-usd`).

e.g
` Error: Failed verification: ' image ' -- Coding Error in _LoadTexture at line 422 of maya-usd\lib\mayaUsd\render\vp2RenderDelegate\material.cpp`